ftdc/ftdc.go

```diff
@@ -106,9 +107,17 @@ type FTDC struct {
 	//
 	// When debug is true, the `outputWriter` will "tee" data to both the `currOutputFile` and
 	// `inmemBuffer`. Otherwise `outputWriter` will just refer to the `currOutputFile`.
-	debug          bool
-	outputWriter   io.Writer
-	currOutputFile *os.File
+	debug        bool
+	outputWriter io.Writer
+	// bytesWrittenCounter will count bytes that are written to the `outputWriter`. We use an
+	// `io.Writer` implementor for this, as opposed to just counting by hand, primarily as a
```

Comment:
```suggestion
	// `io.Writer` implementer for this, as opposed to just counting by hand, primarily as a
```

Reply:
Applied locally (quarreling with lint)

Also, any tips for knowing when to use or vs er? We definitely didn't spelling test the nounification of the entirety of verbs in my K-12 education.

Reply:
I love these questions and wish I had a good answer for you. 

Some people say if you can add the "-ate" suffix to form a verb from a stem, you can _further_ add "-or" to form a "agentive" noun (entity that does the action). e.g. "decor" -> "decorate" -> "decorator". While I can't think of any counterexample to that rule, it fails to capture much a lot of other "-or" words. e.g. "destr" -> "destroy" -> "destructor". In reality, I think it's an etymology thing (latin vs. anything else) combined with a this-is-how-it-got-spelled-in-some-old-dictionary-and-now-we-do-that-in-modern-english thing.

Reply:
> In reality, I think it's an etymology thing (latin vs. anything else) combined with a this-is-how-it-got-spelled-in-some-old-dictionary-and-now-we-do-that-in-modern-english thing.

Oh nice, this feels ripe for "regularization". In that the "wrong" spelling is almost always understood. So we can just choose to make all versions acceptable. Or, when in doubt, choose to standardize on "always -er" or "always -or".